build-scripts/build_windows_installer.bat

```diff
@@ -45,6 +45,32 @@ if defined VS160COMNTOOLS (
 if defined VS150COMNTOOLS (
   set TOOL_CHAIN_TAG=VS2017
 )
+
+set CHECK_VS2022=0
+if not defined TOOL_CHAIN_TAG (
+  set CHECK_VS2022=1
+) else (
+  if /I "%TOOL_CHAIN_TAG%"=="VS2022" (
+    set CHECK_VS2022=1
+  )
+)
+if %CHECK_VS2022% NEQ 0 (
+  for /f "usebackq tokens=1* delims=: " %%i in (`"%VS_WHERE%" -version 17.0`) do (

```

Comment:
This should be:

```for /f "usebackq tokens=1* delims=: " %%i in (`"%VS_WHERE%" -version [17.0^,18.0^)`) do (```

That way it all versions that all within 17.0<=n<18.0 are found. So when MSFT releases 17.1 or newer the script still works.